siloLapisTests/test/common.ts

```diff
@@ -23,3 +23,27 @@ export const lapisClient = new LapisControllerApi(new Configuration({ basePath }
     return Promise.resolve(responseContext.response);
   },
 });
+
+export const lapisSingleSegmentedSequenceController = new SingleSegmentedSequenceControllerApi(
+  new Configuration({ basePath })
+).withMiddleware({
+  onError: errorContext => {
+    if (errorContext.response) {
+      console.log('Response status code: ', errorContext.response.status);
+      console.log('Response body: ', errorContext.response.json());
+    }
+    if (errorContext.error) {
+      console.error(errorContext.error);
+    }
+    return Promise.resolve();
+  },
+  post: async responseContext => {
+    if (responseContext.response.status >= 300) {
+      const response = responseContext.response.clone();
+      console.log('Request URL:', responseContext.url);
+      console.log('Response status code:', response.status);
+      console.log('Response body:', await response.json());
+    }
+    return Promise.resolve(responseContext.response);
+  },
+});
```

Comment:
We could also extract the middleware, since it's the same as for the `lapisClient`, isn't it?